streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/SubscriptionInfoTest.java

```diff
@@ -31,13 +33,21 @@
 
 public class SubscriptionInfoTest {
     private final UUID processId = UUID.randomUUID();
-    private final Set<TaskId> activeTasks = new HashSet<>(Arrays.asList(
+    private static final Set<TaskId> ACTIVE_TASKS = new HashSet<>(Arrays.asList(
         new TaskId(0, 0),
         new TaskId(0, 1),
         new TaskId(1, 0)));
-    private final Set<TaskId> standbyTasks = new HashSet<>(Arrays.asList(
+    private static final Set<TaskId> STANDBY_TASKS = new HashSet<>(Arrays.asList(
         new TaskId(1, 1),
         new TaskId(2, 0)));
+    private static final Map<TaskId, Integer> TASK_LAGS = new HashMap<>();
```

Comment:
prop: Just a proposal
```suggestion
    private static final Map<TaskId, Integer> TASK_LAGS = mkMap(
        mkEntry(new TaskId(0, 0), -1),
        mkEntry(new TaskId(0, 1), -1),
        mkEntry(new TaskId(1, 0), -1),
        mkEntry(new TaskId(1, 1), 0),
        mkEntry(new TaskId(2, 0), 10)
    );
```